style.css

```diff
@@ -1,4 +1,74 @@
 body {
-background-color: #232323
+  color: #ffffff;
+  font-family: 'Ubuntu', sans-serif;
+  font-size: 15px;
+  width: auto;
+  background: url(nieckovsky_image_main.jpg);
+  background-color: #111415;
+  background-size: 100%;
+  background-position: center;
+  background-attachment: fixed;
+  background-repeat: no-repeat;
+}
+
+header {
+  width: 100%;
+  height: 50px;
+}
+
+nav {
+  display: inline-block;
+  width: 100%;
+  position: relative;
+  margin-top: auto;
+  margin-bottom: auto;
+}
+
+span {
+  display: inline-block;
+  width: 125px;
+  height: 50px;
+  background-image: url(nieckovsky_logo_top.png);
+  background-size: 125px 50px;
+  background-repeat: no-repeat;
+  font-size: 0;
+  line-height: 0;
+  vertical-align: middle;
+  position: absolute;
+  margin-top: auto;
+  margin-bottom: auto;
+  margin-left: 5%;
+}
+
+ul {
+  list-style-type: none;
+  vertical-align: middle;
+  height: 26px;
+  margin-left: 40%;
+  margin-top: 12px;
+  margin-bottom: 12px;
+}
+
+li {
+  display: inline-block;
+  vertical-align: text-top;
```

Comment:
this vertical-align shouldn't be needed with `ul` without its margins and correctly used `display: inline-block` with `vertical-align: middle`